src/reactviews/pages/TableDesigner/designerResultPane.tsx

```diff
@@ -119,67 +111,116 @@ export const DesignerResultPane = () => {
-            renderHeaderCell: () => <>{locConstants.tableDesigner.severity}</>,
-        }),
-        createTableColumn({
-            columnId: "description",
-            renderHeaderCell: () => (
-                <>{locConstants.tableDesigner.description}</>
-            ),
-        }),
-        createTableColumn({
-            columnId: "propertyPath",
-            renderHeaderCell: () => <></>,
-        }),
-    ];
-    const [columns] =
-        useState<TableColumnDefinition<DesignerIssue>[]>(columnsDef);
-    const items = metadata?.issues ?? [];
+    const openAndFocusIssueComponet = async (issue: DesignerIssue) => {
+        const issuePath = issue.propertyPath ?? [];
+        console.log(`focusing on`, issuePath);
+        if (!metadata?.view?.tabs || !state?.provider) {
+            return;
+        }
+        const containingTab = metadata.view.tabs.find((tab) => {
+            return tab.components.find((c) => {
+                return c.propertyName === issuePath[0];
+            });
+        });
 
-    const sizingOptions: TableColumnSizingOptions = {
-        severity: {
-            minWidth: 50,
-            idealWidth: 50,
-            defaultWidth: 50,
-        },
-        description: {
-            minWidth: 500,
-            idealWidth: 500,
-            defaultWidth: 500,
-        },
-        propertyPath: {
-            minWidth: 100,
-            idealWidth: 100,
-            defaultWidth: 100,
-        },
-    };
+        if (!containingTab) {
+            return;
+        } else {
+            state.provider.setTab(containingTab.id as any);
+            await new Promise((resolve) => setTimeout(resolve, 100));
+        }
+        let tableComponent;
+        let tableModel;
+        if (issuePath.length > 1) {
+            // error is found in a table row. Load properties for the row
+            tableComponent = containingTab.components.find(
+                (c) => c.propertyName === issuePath[0],
+            );
+            if (!tableComponent) {
+                return;
+            }
+            tableModel = metadata.model![tableComponent.propertyName];
+            if (!tableModel) {
+                return;
+            }
+            state?.provider.setPropertiesComponents({
+                componentPath: [issuePath[0], issuePath[1]],
+                component: tableComponent,
+                model: tableModel,
+            });
+        }
 
-    const [columnSizingOption] =
-        useState<TableColumnSizingOptions>(sizingOptions);
-    const { getRows, columnSizing_unstable, tableRef } = useTableFeatures(
-        {
-            columns,
-            items: items,
-        },
-        [
-            useTableColumnSizing_unstable({
-                columnSizingOptions: columnSizingOption,
-            }),
-        ],
-    );
-    const rows = getRows();
+        let elementToFocus: HTMLElement | undefined = undefined;
+        switch (issuePath.length) {
+            case 1:
```

Comment:
Lots of magic constants here... What do these numbers represent?

At a minimum, they need comments, but making them named constants would be even better.